cmd/apps.go

```diff
@@ -16,9 +16,26 @@ import (
 	"github.com/deis/workflow-cli/settings"
 )
 
+// AppCmd is interface definition for running app commands
+type AppCmd interface {
```

Comment:
`Commander` I like it! 😃

I was thinking that have a few smaller interfaces might be easier to work with so I could have several smaller mocks  rather than 1 larger one. It looks like it would be the same amount of code either way, so I erred on the side of smaller interfaces.

What do you think?